__tests__/index.spec.ts

```diff
@@ -52,3 +52,21 @@ it('should accept custom config', async () => {
   expect(news[0]).has.property('url').which.is.a('string')
   expect(news[0]).has.property('date').which.is.a('date')
 })
+
+it('filte date', async () => {
+  const news = await getNews({
+    region: 'na',
+    category: 'topics',
+    count: 100,
+    after: new Date(1656633600 * 1000),
```

Comment:
Add a comment explains that what time is this.
And it is better to test within a shorter duration.